src/gui/application.cpp

```diff
@@ -425,13 +412,17 @@ void Application::slotAccountStateAdded(AccountStatePtr accountState) const
     connect(accountState.data(), &AccountState::stateChanged,
         _gui.data(), &ownCloudGui::slotAccountStateChanged);
     connect(accountState->account().data(), &Account::serverVersionChanged,
-        _gui.data(), &ownCloudGui::slotTrayMessageIfServerUnsupported);
+        _gui.data(), [account = accountState->account().data(), this] {
+            _gui->slotTrayMessageIfServerUnsupported(account);
+        });
```

Comment:
I do not get why this implementation is better than the old one.

Reply:
I had to touch this connect as I changed the signature of the signal.
the old emit serverVersionChanged(this).
It is highly uncommon for Qt signal to pass this with the emit.